app/controllers/admin/invoices_controller.rb

```diff
@@ -0,0 +1,3 @@
+class Admin::InvoicesController < AdminController
```

Comment:
I am wondering if it's a good idea to continue creating files/methods we don't yet need. I get being prepared, but looking at all the methods created in some of the other controllers I don't think it was necessary.